backend/store/queue.go

```
@@ -0,0 +1,162 @@
+func (ss QueueStore) GetNextSong(queueId uuid.UUID) (*models.Song, error) {
+	song := &models.Song{}
+	err := ss.db.Get(song, "select s.* from song s where s.song_id = (select qs.song_id from queue_song qs where qs.queue_id = $1 order by qs.position asc offset 1 limit 1)", queueId)
+	return song, err
+}
+
+func (ss QueueStore) PlayNextSong(queueId uuid.UUID) (*models.Song, error) {
+	tx, err := ss.db.Beginx()
+	if err != nil {
+		return nil, err
+	}
+	defer tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)